README.md

```diff
@@ -45,3 +45,17 @@ from parametrization import Parametrization
 def test_somthing(actual, expected):
     assert actual == expected
 ```
+### Generate name based on arguments
+```python
+from parametrization import Parametrization
+
+@Parametrization.name_factory(lambda actual, expected: '{}=={}'.format(a, b))
```

Comment:
you example has an error, you have to cboose if you call your arguments `a` & `b` or `actual` & `expected` but not both.

In regard of the rest of the code, you can fix/reformat like this:
```py
@Parametrization.name_factory(lambda actual, expected: f'{actual}=={expected}')
```

Reply:
Readme is now fixed.

Do you want to get the tests/test_parametrization.py changed also to use actual and expected even though the other tests use a and b as names?

Reply:
..and about using f-strings. Should I also update setup.py and drop support for Python versions older than 3.6 (as f-strings were introduced in Python 3.6)?

Reply:
hey @janneronkko and @yoyonel :) 
Thanks for the great PR! 
We want to keep Python2.7 support for now since we're using the package in different code versions. 

We'll go ahead and merge your changes.